utils/demarchesSimplifiees.js

```diff
@@ -13,18 +13,15 @@ exports.DOSSIER_STATE = {
   sans_suite: 'sans_suite',
 };
 
-function getChampValue(champData, attributeName, stringValue = true) {
+function getChampValue(champData, attributeName) {
   const potentialStringValue = champData.find((champ) => champ.label === attributeName);
 
   if (typeof potentialStringValue === 'undefined') {
     console.warn(`Champ from API ${attributeName} does not exist`);
-
     return '';
   }
-  if (stringValue) {
-    return potentialStringValue.stringValue.trim();
-  }
-  return potentialStringValue.value.trim();
+
+  return potentialStringValue.stringValue.trim();
```

Comment:
Why do we never return value anymore?

Reply:
because it's not part of the doc https://demarches-simplifiees-graphql.netlify.app/champ.doc.html